geode-core/src/test/java/org/apache/geode/distributed/internal/DistributionConfigJUnitTest.java

```diff
@@ -104,7 +104,7 @@ public void before() {
   @Test
   public void testGetAttributeNames() {
     String[] attNames = AbstractDistributionConfig._getAttNames();
-    assertThat(attNames.length).isEqualTo(169);
+    assertThat(attNames.length).isEqualTo(170);
```

Comment:
This syntax provides better failure messages:
```
assertThat(attNames).hasSize(170);
```
I'd go ahead and convert the test to use only AssertJ. There's an IntelliJ plugin that can automate this but it's not very smart, I usually end up having to touch up anything involving collections, arrays, or messages.